.travis.yml

```diff
@@ -173,7 +173,7 @@ before_script:
       && scripts/create_source_tarball.sh)
 
 script:
-    - test $SOLC_EMSCRIPTEN != On || (scripts/test_emscripten.sh)
+    - test $SOLC_EMSCRIPTEN != On -o $SOLC_TESTS != On || (scripts/test_emscripten.sh)
```

Comment:
Maybe we should have another flag "temporarily enable full travis test suite" to turn this and the other thing on or off :)

Reply:
The tests are just 100% broken on travis. They take too long.